### DISCUSSION TITLE
MY ACCOUNT IS NOT SHOWN

### DISCUSSION THREAD

**Discussion by heisenberg3376** [community/community#60399] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please! Somebody help me out. When I type my github profile url in incognito tab, it is saying "404 error this is not the web page you are looking for". Please help me anyone what should I do??

**Comment by cehfisher** [comment]
Hi @heisenberg3376 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
